### ISSUE TITLE
[Suggestion] Right-clicking on icons to switch on/off modules

### ISSUE THREAD

**Issue by EzerArch** [MachineMuse/MachineMusePowersuits#785]
_(In reference to #589, but as a simpler way)_

Until now, there are two ways to switch on/off modules:
* by keybind.
* by removing them (not much of help, but anyway).

Since I can right-click on module keybind icons in the keybind GUI to add them to the HUD, what if I could right-click the module icon to switch it on and off so I don't need to create a keybind for all modules?

![2017-04-17_02 41 22](https://cloud.githubusercontent.com/assets/6743303/25079975/5991b866-2317-11e7-8066-c1ac0ea9d8ed.png)

If you right-click a module icon, a small green "OFF" is added (or the icon can be grayed out) to tell if the module is activated or not.

**Comment by eyeonus** [comment]
Just so you know, every one of those modules you have bound to ] in the screenshot are on by default.

**Comment by EzerArch** [comment]
> are on by default.

Yes, until I press [ a few times or by accident and I have to test or add them to HUD to know.

But my idea is that you wouldn't need to keybind the modules to switch them off, and add to HUD to know if a module is silently draining energy.

![2017-04-17_03 48 25](https://cloud.githubusercontent.com/assets/6743303/25081427/ba02ff08-2320-11e7-8c9a-476f1ca972f3.png)

Additionally, I don't need to use Magnet, Active Camo and Mob Repulsor all the time. I could give each a keybind of its own but I keep running out of keys. I can't just hook them to a single keybind because I don't need to use them together and it'd be overkill energy-wise.

Keybinding 10 modules (that I seldom need to change) to a single, "weird" key was a workaround.

**Comment by lehjr** [comment]
This would require having a tinker table nearby. I could make the portable tinker table a full table and that would help, but I am curious how players normally deal with running out of keybinds.
